gh-pages/playground.html

```
@@ -0,0 +1,149 @@
+</style>
+
+<div class="playground-container">
+  <h1>Sarek Transpiler Playground</h1>
+  <p class="subtitle">
+    Write an OCaml kernel expression and see the GPU source output.
+    The transpiler runs entirely in your browser.
+  </p>
+
+  <div class="editor-pane">
+    <label for="kernel-source">Kernel source</label>
+    <textarea id="kernel-source">fun (a : float32 vector) (b : float32 vector) -> let i = global_thread_id in b.(i) <- Float32.sin a.(i)</textarea>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)